protocol/daemons/server/pricefeed.go

```diff
@@ -52,16 +45,10 @@ func (s *Server) UpdateMarketPrices(
 		metrics.Latency,
 	)
 
-	// If the daemon is unable to report a response, there is either an error in the registration of
-	// this daemon, or another one. In either case, the protocol should panic.
-	if err := s.reportResponse(servertypes.PricefeedDaemonServiceName); err != nil {
-		panic(err)
-	}
-
 	if s.marketToExchange == nil {
```

Comment:
nit: add comment why we would still panic here.

Reply:
This panic is removed.

Reply:
Ahhh, IC. It looks to me like this panic would only occur if the client pinged the server before the prices cache was initialized on the server.

Reply:
gotcha, can you add some comments that "This is unexpected because xyz"